### PR TITLE
Profile skeleton

### PR DESCRIPTION
**Pull request by eatonphil**
screen shot 2017-01-26 at 11 39 09 am

### CODE REVIEW

**Comment by eatonphil** [comment]
Closes #1079

File: src/profile/layouts/AuthenticationPage.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+                <FormGroupError errors={errors} name="password" />
+              </div>
+            </FormGroup>
+            <FormGroup className="row" errors={errors} name="expires">
+              <div className="col-sm-2 label-col">
+                <label>Expires:</label>
+              </div>
+              <div className="col-sm-10">
+                <Select onChange={expires => this.setState({ expires })} value={expires}>
+                  <option value="0">Never</option>
```
Comment (mparke) [inline src/profile/layouts/AuthenticationPage.js]: Couple questions re: expiry:

Assuming the api does support this in the future, will we be fetching expiry options?

Have we thought about the process of handling a soon-to-expire password in the UI? We might want to hide + disable this until we have a solution.

Comment (eatonphil) [inline src/profile/layouts/AuthenticationPage.js]: > Assuming the api does support this in the future, will we be fetching expiry options?

I don't think so.

> Have we thought about the process of handling a soon-to-expire password in the UI? We might want to hide + disable this until we have a solution.

See [alerts](https://github.com/404). :)

**Comment by eatonphil** [comment]
I think I forgot to disable the expires field.

**Comment by eatonphil** [comment]
Nope, I took care of it.

**Comment by eatonphil** [comment]
Thanks!

**Pull request merged**